### ISSUE TITLE
Advanced Topics in Software Verification

### ISSUE THREAD

**Issue by lsf37** [fme-teaching/fm-courses#61]
Course code, if applicable: COMP4161
University hosting the course: UNSW Sydney
Contact person: Gerwin Klein, kleing@unsw.edu.au
Concepts taught: Applied Formal Methods,Axiomatic Semantics ,Deductive Software Verification,Deductive Proof,Deductive Verification ,Formal Specification,Hoare Logic,Induction,Interactive Theorem Proving,Logic,Proofs,Proof Assistants,Software Verification,Specification,Structural Operational Semantics,Theorem Proving,WP Calculus
Tools used: C,Isabelle
Webpage: http://www.cse.unsw.edu.au/~cs4161/
Year/Level: 3
Reviewed: 2021/06/25

**Comment by luigiapetre** [comment]
The webpage provided here (https://www.cse.unsw.edu.au/~cs4161/) is not the same as the one in the active database (https://fme-teaching.github.io/courses/), where for some reason it opens this address http://www.cse.unsw.edu.au:443/~cs4161/, that does not work.

**Comment by jff** [comment]
@luigiapetre Fixed!

**Comment by luigiapetre** [comment]
@jff:  but it does not show the "reviewed" field, maybe because it's not "Last review date"?

**Comment by luigiapetre** [comment]
@jff: And in my browser it still does not work (the link is still http://www.cse.unsw.edu.au:443/~cs4161/)

**Comment by lsf37** [comment]
If it helps, a non-https link should also be fine, i.e. http://www.cse.unsw.edu.au/~cs4161/

**Comment by jff** [comment]
@luigiapetre I added the **Reviewed** field. 

The link was working on Brave but not in Firefox (not sure why). I followed @lsf37 's suggestion and now it seems to be working as expected!

**Comment by lsf37** [comment]
I am no longer teaching this course at UNSW, [Thomas Sewell](https://trustworthy.systems/people/?cn=Thomas+Sewell) (@talsewell) is now the lecturer in charge of it.

**Comment by talsewell** [comment]
Yes, I'm currently the responsible lecturer, and I'm happy to be listed in this list or database or whatever it is.
